examples/json/README.md

```diff
@@ -0,0 +1 @@
+# json
```

Comment:
@Jagl257 Could you please add a little bit more of description?

Reply:
Ok @CristhianMotoche, can you check them out, please?